### PR TITLE
Aadvanced search styling

### PR DESCRIPTION
**Pull request by berachele**
# Description
Added spacing to the filter buttons to be more spread out, bigger font with buttons as well as results. Switched out button to be an Addition icon. Icon made it so that you can't really tell if the button's been click (and if it's really added to the selected cities list) but it is still working. Trying to work on a hover type effect for it.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Change Status
- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback
## Has This Been Tested
- [ ] Yes
- [x] No
- [ ] Not necessary
## Does this require communication between DS and WEB?
- [ ] Yes, and I have already reached out to someone
- [ ] Yes, but I still need to reach out to someone
- [x] No
## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least two peers
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/AdvancedSearch/SearchResult.js [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
   return (
     <div className="search-result">
-      {`${name}, ${state}`}
-      <Button onClick={addCityToComparison}>Add to Comparison</Button>
+      {`${name}, ${state}`}&nbsp;&nbsp;
+      <img
+        src={addIcon}
+        alt="circle with plus sign in middle icon"
+        onClick={addCityToComparison}
+        onMouseOver={addIconHover}
```
Comment (ddhorstman) [inline src/components/pages/AdvancedSearch/SearchResult.js:17]: This crashes the app on hover :(

**Review by ddhorstman — COMMENTED** [review]
Looks good so far! It looks like you don't have some of the latest styling changes Alan made on this branch - we'll have to keep that in mind when merging.

As of right now, the hover effect on the image to add to comparison crashes the app. I think you need to add some more logic there, maybe something with useState or something like what Lyndsi did in the AboutPage.

**Review by ddhorstman — COMMENTED** [review]
Got it!

**Review by alanblee — APPROVED** [review]
nice job!

**Pull request merged**